Client/Client.Qt/Widgets/LoadingSpinner.hpp

```diff
@@ -47,10 +47,9 @@ private slots:
     /// Put on center of page
     void updatePosition();
 
-private:
     const Style::LoadingSpinner& _st;
     std::unique_ptr<QTimer>      _timer;
     bool                         _disableParentWhenSpinning;
-    int32_t                      _currentCounter;
-    bool                         _isSpinning;
+    uint16_t                     _currentCounter = 0;
+    bool                         _isSpinning = false;
```

Comment:
I think, equal signs should be set according code styling (like in ch. "Formatting", p.9)